numpy/core/tests/test_multiarray.py

```diff
@@ -7196,6 +7197,21 @@ def test_relaxed_strides(self):
                     arr, ['C_CONTIGUOUS'])
             assert_(strides[-1] == 8)
 
+    @pytest.mark.valgrind_error(reason="leaks buffer info cache temporarily.")
+    @pytest.mark.skipif(not np.ones((10, 1), order="C").flags.f_contiguous,
+            reason="Test is unnecessary (but fails) without relaxed strides.")
```

Comment:
why does this fail without relaxed strides?

Reply:
The array will be only C contiguous, so you can't request an f-contiguous buffer.

Reply:
Makes sense, thanks. Am I right in thinking code-coverage skips this test, based on the review comment it made in the diff?

Reply:
That seems strange, unfortunately the comment is gone now due to the small comment change, so have to see which branch is uncovered and whether it should be later.

Reply:
Maybe the uncovered pass was within the `else` block of `#if NPY_RELAXED_STRIDES_CHECKING`? That can't be hit, since codecov should be running in the default setup.

Reply:
Well, the codecoverage didn't lie, the code got messed up in a cleanup try... The test was missing the strides check to notice it :(.

anyway, fixed now...